ctx_test.go

```diff
@@ -54,6 +54,9 @@ func TestCtxSetDefaultVerifyLocations(t *testing.T) {
 	}
 
 	conn, err := Dial("tcp", "google.com:443", ctx, 0)
+	if conn == nil || err != nil {
+		t.Skip("can't connect, skipping the test.")
```

Comment:
I am skipping the test now with a Debian-only patch in the deb package repo.

Here we should not skip the test. If any, we can add a test failure if the test cannot connect.